README.md

```diff
@@ -387,6 +420,20 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **How to sign up**: Check out the [Mattermost website](https://mattermost.com/blog/hacktoberfest-2021/) for more details
 - **Notes**: Check out the [Mattermost handbook for Hacktoberfest](https://handbook.mattermost.com/contributors/contributors/ways-to-contribute/hacktoberfest2021) and their [community colab server](https://docs.mattermost.com/guides/community-chat.html)
 
+#### **Milvus**
+
+- **Swag**: Logitech Pro X Keyboard, a DJI Drone, a sticker pack, a digital Milvus contributor badge, and a Milvus T-shirt (For 2+ PRs)
+- **Requirements**: Sign up for the event on the [Hacktoberfest website](https://milvus.io/hacktoberfest-2021/)  and just start contributing to any open source project - it’s as simple as that! If you’re new to the open source world, no worries, we’ve got you covered with [Guide](https://discuss.milvus.io/t/join-hacktoberfest-2021-with-us/72).
+  - By contributing to the Milvus project between October 1 and October 31 you’ll also earn some additional swag and prizes sponsored by Zilliz
+  - Swag tiers
+    - **For Top Contributor (Best Quality PR)** : A Logitech Pro X Keyboard and a physical Hacktoberfest badge.
+    - **For Top Contributor (Most Difficult PR) :trophy:** : A DJI Drone if you complete at least 1 PR merged with the issue tagged #advanced .
+- **How to sign up**:
+  - Sign up [here](https://milvus.io/hacktoberfest-2021/)
+  - Join Discussions by creating [Milvus Forum Account](https://discuss.milvus.io/c/hacktoberfest/9) and interact with community members.
+- **Issues**: Issues are stated at Issues Section of [Hacktoberfest website](https://milvus.io/hacktoberfest-2021/).
+- **Notes**: Check out [Milvus Community Forum](https://discuss.milvus.io/c/hacktoberfest/9)
+
```

Comment:
Following the [Contributing guidelines](https://hacktoberfestswaglist.com/contributing), I'd love to see you format this entry again. Use the examples from line 280-287 above.